packages/stdlib/test.pony

```diff
@@ -30,6 +30,7 @@ use logger = "logger"
 use math = "math"
 use net = "net"
 use options = "options"
+use persistent = "collections/persistent"
```

Comment:
I think it makes sense to name this package label `collections_persistent` (and move it up to below the `collections` line above).

This would keep from clashing with the name of a `persistent` package later if we add it (maybe it would be a package about storing data on disk?).  Even if we never create such a package, I think it would be better to be consistent here and use the full package name.